assets/notifications/reducers.js

```diff
@@ -15,7 +15,7 @@ export default function notificationReducer(state = initialState, action) {
     switch (action.type) {
 
     case NEW_NOTIFICATION: {
-        const notifications = state.notifications ?
+        const notifications = state.notifications.length ?
```

Comment:
this could be also `const notifications = notifications.concat([action.notification.item]);`